assets/js/modules/idea-hub/components/dashboard/DashboardIdeasWidget/index.js

```diff
@@ -193,26 +193,26 @@ function DashboardIdeasWidget( props ) {
 							'Idea Hub is generating ideas',
 							'google-site-kit'
 						) }
-						subtitle={ __(
-							'This could take 24 hours.',
-							'google-site-kit'
-						) }
 					/>
 				</div>
 			</Widget>
 		);
 	}
 
-	const WrappedFooter = () => (
-		<Footer
-			tab={ activeTab }
-			content={
-				activeTab === 'new-ideas'
-					? __( 'Updated every 2-3 days', 'google-site-kit' )
-					: false
-			}
-		/>
-	);
+	let WrappedFooter;
+	if ( activeTab === 'new-ideas' ) {
+		WrappedFooter = () => (
+			<Footer
+				tab={ activeTab }
+				footerText={ __( 'Updated every 2-3 days', 'google-site-kit' ) }
```

Comment:
Just a question: why use a special prop here instead of `children`? Eg. `<Footer tab={ activeTab }>{ __( 'Updated every 2-3 days', 'google-site-kit' ) }</Footer>`

Not a big deal either way, but using `children` here seems fine 🤷🏻‍♂️